BlueprintUI/Sources/Element/ProxyElement.swift

```diff
@@ -15,11 +15,10 @@ public protocol ProxyElement: Element {
 extension ProxyElement {
```

Comment:
Curious for your thoughts on this @watt @kylebshr @bencochran; this change alone seems to give us a ~25% boost with deep hierarchies, without any other issues that I can find; since it short circuits through to the proxy content vs. passing through the `ElmementContent` machinery.

Reply:
🤷  I can't think of anything this would break offhand.

Can you elaborate on the benchmarks you used to see a 25% improvement? What was the absolute time? Is that for layout? A whole render pass?

Reply:
It was from running either of these two tests in here: https://github.com/square/Blueprint/pull/200/files#diff-7089446d6d4f9cc1ab8538880005177c61977553e2bdf6341300e7f5d184c177

Which basically just in a tight loop re-apply that hierarchy to a blueprint view. I think it would be helpful to build out a more realistic perf toolset, so I will take a look at expanding on that file a bit in here.

Reply:
Full results:

```
Before
---------

Iterations: 48, Average Time: 0.2101164162158966
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.143 seconds).

Iterations: 55, Average Time: 0.18392309058796275
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.289 seconds).


ProxyElement Changes
---------------------

Iterations: 66, Average Time: 0.15314857526258988
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.164 seconds).

Iterations: 61, Average Time: 0.1642437391593808
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.209 seconds).


Plus Label Changes
-------------------

Iterations: 73, Average Time: 0.1379784933508259
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.125 seconds).

Iterations: 67, Average Time: 0.1496717751915775
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.233 seconds).
```

Reply:
And then in `RELEASE`:

```
Before
---------

Iterations: 113, Average Time: 0.0890231501739637
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.128 seconds).

Iterations: 78, Average Time: 0.12875753793960962
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.292 seconds).


ProxyElement Changes
---------------------

Iterations: 175, Average Time: 0.05717774868011474
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.084 seconds).

Iterations: 90, Average Time: 0.11165261136160956
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.332 seconds).


Plus Label Changes
-------------------

Iterations: 210, Average Time: 0.0477486667178926
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_deep_element_hierarchy]' passed (10.112 seconds).

Iterations: 95, Average Time: 0.10646694710380153
Test Case '-[BlueprintUICommonControls_Unit_Tests.PerformanceTests test_repeated_layouts]' passed (10.395 seconds).
```